.github/workflows/release.yaml

```diff
@@ -54,13 +58,33 @@ jobs:
           RELEASE: ${{ env.RELEASE_VERSION }}-ARM
         run: |
           cd opensearch-operator
-          pwd
-          ls
-          go version
-          go env
           aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/opsterio
           docker build -f Dockerfile-ARM -t public.ecr.aws/opsterio/opensearch-operator:$RELEASE ./
           docker push public.ecr.aws/opsterio/opensearch-operator:$RELEASE
           docker tag public.ecr.aws/opsterio/opensearch-operator:$RELEASE public.ecr.aws/opsterio/opensearch-operator:latest-ARM
           docker push public.ecr.aws/opsterio/opensearch-operator:latest-ARM
-
+      - name: Copy-crd-to-chart-dir
+        run: |
+          cd opensearch-operator
+          make manifests
+          cp ./config/crd/bases/opensearch.opster.io_opensearchclusters.yaml ../charts/opensearch-operator/templates/
+          mv ../charts/opensearch-operator/templates/opensearch.opster.io_opensearchclusters.yaml ../charts/opensearch-operator/templates/opensearchclusters.opensearch.opster.io-crd.yaml
+          sed -i '' -e 's/^version.*/version: $RELEASE_VERSION/' charts/opensearch-operator/Chart.yaml
```

Comment:
version is the version of the chart, so we don't want to change that here, instead we want to set the appVersion and replace the `latest` tag from `values.yaml` with the RELEASE_VERSION. `values.yaml` then also needs to be added to the commit.

Reply:
So we need to change command should be :
` sed -i '' -e 's/^ appVersion.*/appVersion: $RELEASE_VERSION/' charts/opensearch-operator/Chart.yaml ` ?
and after that 
`sed -i '' -e 's/^ tag.*/tag: $RELEASE_VERSION/' opensearch-k8s-operator/charts/opensearch-operator/values.yaml` ?

Reply:
With the corrct escaping and all the commands should be:
`sed -i -e 's/^appVersion.*/appVersion: '$RELEASE_VERSION'/' charts/opensearch-operator/Chart.yaml`
`sed -i -e 's/tag.*/tag: '$RELEASE_VERSION'/' charts/opensearch-operator/values.yaml`